internal/automation-security@-emails/script.conf

```diff
@@ -0,0 +1,6 @@
+userID = <PlaceUserIDHere>
```

Comment:
Better to add a small description explaining the content of the file as a comment.

Reply:
Done!